runtime/src/snapshot_utils.rs

```diff
@@ -1999,7 +1997,9 @@ pub fn verify_snapshot_archive(
 ) {
     let temp_dir = tempfile::TempDir::new().unwrap();
     let unpack_dir = temp_dir.path();
-    let unpack_account_dir = create_accounts_run_and_snapshot_dirs(unpack_dir).unwrap().0;
+    let unpack_account_dir = accounts_db::create_accounts_run_and_snapshot_dirs(unpack_dir)
```

Comment:
nit: I've found the tuple indexing to be kind of tricky sometimes. Doing the full de-structure is a bit more explicit:
```rust
let (unpack_account_dir, _snapshot_dir) = ...
```
This isn't perfect either as someone could still swap the order of the return types without breaking anything. I guess a struct with names fields, and unpacking by named field would be the only way to fully protect against this.

In any case, feel free to change or ignore; certainly don't want to punish the good deed of restricting access to the test only function

Reply:
Lol, missed that you added `auto-merge` label. Like I said, I was fine pushing as-is anyways